packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -40,62 +100,225 @@ export class SankeyChartBase extends React.Component<
+          ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
+        >
+          <svg width={width} height={height} id={getId('sankeyChart')}>
+            <g className={this._classNames.links} strokeOpacity={1}>
+              {linkData}
+            </g>
+            <g className={this._classNames.nodes}>{nodeData}</g>
+          </svg>
+        </div>
+      </>
     );
   }
 
+  /**
+   * This is used for calculating the node non normalized value based on link non normalized value.
+   *
+   */
+  private _populateNodeActualValue(data: ISankeyChartData) {
+    data.links.forEach((link: any) => {
+      if (!link.originalValue) {
+        link.originalValue = link.value;
+      }
+    });
+    data.nodes.forEach((node: any) => {
+      node.actualValue = Math.max(
+        d3Sum(node.sourceLinks, (link: any) => link.originalValue),
+        d3Sum(node.targetLinks, (link: any) => link.originalValue),
+      );
+    });
+  }
+
+  /**
+   *
+   * This is used to group nodes by column index.
+   */
+  private _populateNodeInColumns(sankey: d3Sankey.SankeyLayout<d3Sankey.SankeyGraph<{}, {}>, {}, {}>) {
+    const graph = { ...this.props.data.SankeyChartData! };
+    sankey(graph);
+    const nodesInColumn: NodesInColumns = {};
+    graph.nodes.forEach((node: any) => {
+      const columnId = node.layer;
+      if (nodesInColumn[columnId]) {
+        nodesInColumn[columnId].push(node);
+      } else {
+        nodesInColumn[columnId] = [node];
+      }
+    });
+    return nodesInColumn;
+  }
+
+  /**
+   * This is used to normalize the nodes value whose value is less than 1% of the total column value.
+   *
+   */
+  private _adjustOnePercentHeightNodes(
+    sankey: d3Sankey.SankeyLayout<d3Sankey.SankeyGraph<{}, {}>, {}, {}>,
+    nodesInColumn: NodesInColumns,
+  ) {
+    const totalColumnValue = Object.values(nodesInColumn).map((column: any[]) => {
+      return d3Sum(column, (node: any) => node.value);
+    });
+    totalColumnValue.forEach((columnValue: number, index: number) => {
+      let totalPercentage = 0;
+      nodesInColumn[index].forEach((node: any) => {
+        const nodePercentage = (node.value / columnValue) * 100;
+        node.actualValue = node.value;
+        if (nodePercentage < 1) {
+          node.value = 0.01 * columnValue;
+          totalPercentage = totalPercentage + 1;
+        } else {
+          totalPercentage = totalPercentage + nodePercentage;
+        }
+      });
+      const scalingRatio = totalPercentage !== 0 ? totalPercentage / 100 : 1;
+      if (scalingRatio > 1) {
+        nodesInColumn[index].forEach((node: any) => {
+          node.value = node.value / scalingRatio;
+          this._changeColumnValue(node, node.actualValue, node.value);
+        });
+      }
+    });
+  }
+
+  /**
+   *
+   * This is used for normalizing each links value for reflecting the normalized node value.
+   */
+  private _changeColumnValue(node: any, originalValue: number, increasedValue: number) {
```

Comment:
is this increasedValue or decreasedValue

Reply:
yes , my bad, must be decreased value. Will take this up in next commit .